Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -92,10 +92,20 @@ void SubscribeForegroundEventHandler()
 
 int main()
 {
-    // Setup an event handler, so we can receive notifications in the foreground while the app is running.
-    SubscribeForegroundEventHandler();
+    auto pushNotificationManager{ PushNotificationManager::Default() };
 
-    PushNotificationManager::Default().Register();
+    if (pushNotificationManager.IsSupported())
+    {
+        // Setup an event handler, so we can receive notifications in the foreground while the app is running.
+        SubscribeForegroundEventHandler();
+
+        pushNotificationManager.Register();
+    }
+    else
+    {
+        // Here, the app should handle the case where push notifications supported.
```

Comment:
General guidance for applications that don't support Push is to do either one of the following:
1) Maintain their own persistent connection with their App Service
OR
2) Use polling over a scheduled interval to synchronize the client

Can we mention that somewhere in the comment?